praetorian_cli/sdk/test/ui/test_console_tools.py

```
@@ -0,0 +1,197 @@
+    def _wait_for_job(self, *a, **kw):
+        pass
+
+
+def test_run_with_passthrough_executes_locally_when_installed():
+    h = _Harness()
+    with patch('praetorian_cli.runners.local.is_installed', return_value=True), \
+         patch.object(_Harness, '_run_tool_locally') as mock_local:
+        h._cmd_run(['brutus', '10.0.1.5:22', '--protocol', 'ssh', '-U', 'users.txt'])
+
+    assert mock_local.called
+    args, kwargs = mock_local.call_args
```

Ruff (0.15.10)

[warning] Unpacked variable `kwargs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)